Update paper.md (#31)
brmather/pycurious: paper/paper.md

# Summary

Multiple geophysical methods have been proposed to resolve the thermal structure of the Earth's lithosphere with varying degrees of precision.
Geotherms may be constructed from heat flow or xenolith data, but the spatial coverage of these are often limited.
Seismic velocity has proven effective to infer upper-mantle temperature, but its application relies on building a compositional model suitable for the geological context and estimating attenuation from grainsize and water content.
In contrast, magnetic data are among the most widespread geophysical datasets available on the surface of the Earth and can offer useful insight into its thermal structure. 

The magnetic anomaly is the observed magnetic field less the contribution from the core and external fields.

> Geologic features enhance or depress the local magnetic field, but none more so than the concentration of magnetite in the Earth's crust.

Author: ```suggestion
Geologic features enhance or depress the local magnetic field; this response is primarily controlled by the concentration of magnetite in the Earth's crust.
```

> Magnetite is the most prevalent magnetic mineral, in terms of both susceptibility and concentration, and has a Curie point of 580°C.

Author: ```suggestion
Magnetite is the most prevalent magnetic mineral, in terms of both its magnetic susceptibility and concentration in the crust, and has a Curie point of 580°C.
```

# Summary

Magnetic data is one of the most prevelant types of geophysical datasets available on the surface of the Earth. The Curie depth is the depth at which rocks lose their magnetism. The most prevalent magnetic mineral is magnetite, which has a Curie point of 580°C, thus the Curie depth is often interpreted as the 580°C isotherm. Current methods to derive the Curie depth first compute the (fast) Fourier transform over a square window of a magnetic anomaly that has been reduced to the pole. The depth and thickness of magnetic sources is estimated from the slope of the radial power spectrum.

> Our Python package, `pycurious`, ingests a maps of the Earth's magnetic anomaly and distributes the computation of Curie depth across multiple CPUs. `pycurious` implements the [@Tanaka1999] and [@Bouligand2009] methods for computing the thickness of a buried magnetic source. The former selects portions of the radial power spectrum in the low and high frequency domain to compute the depth of magnetic sources, while the latter fits an analytical solution to the entire power spectrum. We cast the [@Bouligand2009] method within a Bayesian framwork to estimate the uncertainty of Curie depth calculations. Common computational workflows and geospatial manipulation of magnetic data are covered in the Jupyter notebooks bundled with this package. The `mapping` module includes a set of functions that help to wrangle maps of the magnetic anomaly into a useable form for `pycurious`. Such an approach is commonly encountered for transforming global compilations of the magnetic anomaly, e.g. EMAG2 [@Meyer2017], from latitudinal/longitudinal coordinates to a local projection in eastings/northings.

Author: ```ingests a maps of the Earth's magnetic anomaly```

is a bit unclear, by "magnetic anomaly" are you referring to a map which has had the IGRF removed? Would you mind clarifying this?

Reviewer: I believe the phrase "magnetic anomaly" implicitly means the total magnetic field less the core and external field contributions:

__anomaly field = observed field - (core field + external fields)__

This has since been clarified in the text.
